packages/glimmer-compiler/lib/template-compiler.ts

```diff
@@ -10,6 +10,10 @@ export interface CompileOptions {
   moduleName?: string;
 }
 
+function isTrustingValue(value) {
```

Comment:
another way to think about it is operation vs value, you create a trusting operation like trusting append, and then there are trusted values that have been prechecked or asserted to have been